src/core/ddsi/src/q_receive.c

```diff
@@ -3468,12 +3468,12 @@ void trigger_recv_threads (const struct ddsi_domaingv *gv)
         ddsrt_iovec_t iov;
         iov.iov_base = &dummy;
         iov.iov_len = 1;
-        GVTRACE ("trigger_recv_threads: %d single %s\n", i, ddsi_locator_to_string (buf, sizeof (buf), dst));
+        GVTRACE ("trigger_recv_threads: %"PRId32" single %s\n", i, ddsi_locator_to_string (buf, sizeof (buf), dst));
```

Comment:
Better use `PRIu32` here. `i` is an `uint32_t`.

Reply:
Good point! I didn't look further than the `%d` in the old format string